app/gui/src/project-view/components/GraphEditor/widgets/WidgetFunction/widgetFunctionCallInfo.ts

```diff
@@ -111,11 +112,14 @@ export function useWidgetFunctionCallInfo(
       Ast.TextLiteral.new(JSON.stringify(args)).code(),
     ]
 
+    const modulePath = project.moduleProjectPath.value
+    const moduleFqn = projectNames.serializeProjectPathForBackend(modulePath)
+
     const expressionId = widgetQuerySubjectExpressionId.value
     if (expressionId != null) {
       return {
         expressionId,
-        visualizationModule: WIDGETS_ENSO_MODULE,
+        visualizationModule: moduleFqn,
```

Comment:
The `else` branch of the enclosing condition probably needs the same change to `visualizationModule`, right?